content/it/_index.md

```diff
@@ -0,0 +1,25 @@
+
+---
+title: "Glossario Cloud Native"
+---
+
+# Glossario Cloud Native
+
+Il Glossario Cloud Native è un progetto guidato dal Business Value Subcommittee (BVS) della CNCF con cui la community si propone di spiegare concetti del mondo cloud native con un linguaggio semplice e diretto che non presupponga conoscenze tecniche pregresse. La [versione in PDF può essere visualizzata e scaricata qui] (https://github.com/cncf/glossary/blob/main/cloudnative-glossary.pdf).
+
+
+## Contribuire
+
+L'invito a suggerire modifiche, aggiunte e migliorie al Glossario Cloud Native è aperto a chiunque abbia voglia di contribuire. Il processo che adottiamo per lo sviluppo e l'arricchimento di questo dizionario condiviso è guidato dalla community e governato dalla CNCF. Il Glossario fornisce una piattaforma vendor-neutral per organizzare un vocabolario condiviso riguardante le tecnologie cloud native. 
+Chiunque voglia contribuire può aprire una issue o una pull request su GitHub. Leggi la sezione [Come Contribuire](/contribuire/) per maggiori informazioni e assicurati di seguire le indicazioni della [Style Guide](/style-guide/).
```

Comment:
I think a space is missing here or did you want it to be in one paragraph. Adding the text in the next line will still show it as one paragraph. It works as one as well but wanted to point it out. Up to you really :)